src/Appwrite/Platform/Tasks/SDKs.php

```
@@ -524,33 +528,45 @@ public function action(?string $platform, ?string $sdk, ?string $version, ?strin
+                        git rm -rf --cached . 2>/dev/null && \
+                        git clean -fdx -e .git -e .github 2>/dev/null && \
                         cp -r ' . $result . '/. ' . $target . '/ && \
-                        (test -d /tmp/.github-backup-$$ && cp -rn /tmp/.github-backup-$$/.github . && rm -rf /tmp/.github-backup-$$ || true) && \
+                        (if [ -d /tmp/.github-backup-$$/.github ]; then cp -rn /tmp/.github-backup-$$/.github . 2>/dev/null && rm -rf /tmp/.github-backup-$$; fi) && \
                         git add -A && \
-                        git commit -m "' . $message . '" && \
-                        git push -u origin ' . $gitBranch . '
-                    ');
+                        git commit -m "' . $message . '" --quiet && \
+                        git push -u origin ' . $gitBranch . ' --quiet 2>&1 | grep -E "^(To |   |[0-9a-f]+\\.\\.[0-9a-f]+)" || true
+                    ', $gitOutput, $gitReturnCode);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$gitOutput'. (undefined)

(UnusedLocalVariable)